news/2019-02-22-acknowledging-community-contributors.md

```diff
@@ -0,0 +1,211 @@
+
+Beyond his award-winning smile, HappyStick has proven himself to be an invaluable addition to the osu!team, heading the new osu! Coffee Hour series, generally spending many sleepless nights setting up and streaming World Cup matches, creating tools to ensure it is easier for others to run them, and so much more.
+
+We could say many things about HappyStick, but foremost among them is how thankful we are for his continued contribution.
+
+### juankristal
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/juankristal.jpg)](https://osu.ppy.sh/users/443656)
+
+Something of a living meme among the osu! tournament scene, juankristal has been perpetually blamed for everything and anything that has gone both wrong and right for countless years.
+
+For once, we are delighted to firmly pin the blame on him for the continuation of the osu! World Cup series as a whole, stepping up to lead the INTENSE organization required to schedule, oversee and generally maintain multiple multinational tournaments.
+
+The workload involved in running the osu! World Cups is *immense*, perhaps far greater than most people realize, and juankristal has not only stepped up himself to do it out of a sheer desire to see it done, but has *excelled* at it, allowing the series to continue without so much as a blip.
+
+### WalterToro
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/WalterToro.jpg)](https://osu.ppy.sh/users/5281416)
+
+A mainstay force within the Global Moderation Team, WalterToro is not your ordinary chat moderator. He is *everywhere*, in the osu!wiki scene, in the tournaments scene, and wildly active in the Spanish community to say the least.
+
+From refereeing matches to refereeing people, we are intensely thankful for WalterToro's incredible work ethic and tireless drive to do what needs to be done.
+
+### clayton
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/clayton.jpg)](https://osu.ppy.sh/users/3666350)
+
+From his involvement in the development of the new website, to reviewing countless improvements and additions to the osu!wiki and various newsposts, to assisting with the maintenance and general day-to-day tasks with Project Loved and much more, clayton has done almost everything there is to do, and continues to keep doing it.
+
+Getting things done is what keeps osu! alive and moving, and there is perhaps no better example of this than clayton in recent times. We are supremely thankful for his efforts over the past few years.
+
+### VeilStar
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/VeilStar.jpg)](https://osu.ppy.sh/users/4255720)
+
+A formerly gigantic presence in the forum technical support team, VeilStar has been responsible for the resolution of literally *thousands* of technical issues over the years and has helped similar amounts of people enjoy osu! to its fullest.
```

Comment:
```suggestion
A formerly gigantic presence in the forums' technical support team, VeilStar has been responsible for the resolution of literally *thousands* of technical issues over the years and has helped similar amounts of people enjoy osu! to its fullest.
```
Considering forums are generally plural

Reply:
reads completely wrong to me even if it's probably (maybe) grammatically correct

Reply:
writing it as a possessive like this implies that each forum has its own technical support team, which isn't true

```suggestion
A formerly gigantic presence in the technical support team of the forums, VeilStar has been responsible for the resolution of literally *thousands* of technical issues over the years and has helped similar amounts of people enjoy osu! to its fullest.
```

maybe like this?

Reply:
Is that not possessive as well? I mean it does read better so I wouldn't mind this instead.

Reply:
i'll meet inbetween and omit any mention of the forums